torchreid/optim/lr_scheduler.py

```diff
@@ -16,23 +16,23 @@ def build_lr_scheduler(optimizer, lr_scheduler, base_scheduler, **kwargs):
     return scheduler
 
 def _build_scheduler(optimizer,
-                lr_scheduler='single_step',
-                base_scheduler=None,
-                stepsize=1,
-                gamma=0.1,
-                max_epoch=1,
-                warmup=10,
-                multiplier = 10,
-                first_cycle_steps=10,
-                cycle_mult = 1.,
-                min_lr = 1e-4,
-                max_lr = 0.1,
-                patience = 5,
-                lr_decay_factor= 100):
+                    num_iter,
+                    lr_scheduler='single_step',
+                    base_scheduler=None,
+                    stepsize=1,
+                    gamma=0.1,
+                    max_epoch=1,
+                    warmup=10,
+                    multiplier=10,
+                    first_cycle_steps=10,
+                    cycle_mult=1.,
+                    min_lr=[1e-4],
```

Comment:
These two params are used in `CosineAnnealingCycleRestart` only and there is no way to deduce the actual number of parameter groups from the config, so I suggest keeping them as float numbers.